src/xcode/ENA/ENA/Source/Services/__tests__/Mocks/MockTestStore.swift

```diff
@@ -35,6 +35,7 @@ final class MockTestStore: Store, AppConfigCaching {
 	var exposureActivationConsentAcceptTimestamp: Int64?
 	var exposureActivationConsentAccept: Bool = false
 	var isOnboarded: Bool = false
+	var onboardingVersion: String = ""
```

Comment:
We should `nil` the heck out of these empty strings!

Reply:
Its not optional and has a default value in the real store. So no nillable.

Reply:
Yes, but the store is crap with this empty String…